hub/core/storage/gcs.py

```diff
@@ -123,12 +124,15 @@ def _connect_browser(self):
             GCSDefaultCredsNotFoundError: if application deafault credentials can't be found.
         """
         try:
-            with open(
-                posixpath.expanduser(
+            if platform.system() == "Windows":
```

Comment:
you can alternatively use `if os.name == "nt":`, we use this for pytorch too, would be good to be consistent